Packs/CommonScripts/Scripts/ZipFile/ZipFile.py

```diff
@@ -4,13 +4,36 @@
 
 import re
 import shutil
-import zipfile
-import pyminizip
+import pyzipper
 from os.path import isfile
 
 ESCAPE_CHARACTERS = r'[/\<>"|?*]'
 
 
+def unzip(zip_file_path: str, password: str = None):
+    with pyzipper.AESZipFile(zip_file_path) as zf:
+        zf.pwd = bytes(password, 'utf-8') if password else None
+        zf.extractall()
+
+
+def compress_multiple(file_names: List[str], zip_name: str, password: str = None):
+    compression = pyzipper.ZIP_DEFLATED
+    encryption = pyzipper.WZ_AES if password else None
+
+    with pyzipper.AESZipFile(zip_name, mode='w', compression=compression, encryption=encryption) as zf:
```

Comment:
cant add zf.write(file_name) in this same block?

Reply:
wdym?